testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/policies/ClientPoliciesAdminTest.java

```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
```

Comment:
Nitpick: year. Same for all new files in the PR.

Reply:
+1

Reply:
+1